ui/src/actions/authors.js

```diff
@@ -51,10 +53,13 @@ export function fetchAuthor(recordId) {
   };
 }
 
-function getAuthorPublicationsQuery(authorsState, newQuery) {
+function getAuthorPublicationsQuery(authorsState, newQuery, user) {
+  const userIsCataloger = isCataloger(Set(user.getIn(['data', 'roles'])));
+  const userIsLoggedIn = user.get('loggedIn');
   return {
     ...authorsState.getIn(['publications', 'query']).toJS(),
     ...newQuery,
+    ...(userIsLoggedIn && userIsCataloger ? { size: 100 } : {}),
```

Comment:
`size: userIsCataloger ? 100 : undefined` is more direct, if you don't have any reason to do this.

Reply:
And move this just before the `newQuery`

Reply:
If i do this, then size is completely removed from queries for non-catalogers

Reply:
ah right, that's why it should be `100 : 10` instead of `100 : undefined`, but anyhow keeping like this already not bad.